govcd/openapi_endpoints.go

```diff
@@ -146,6 +151,14 @@ var endpointElevatedApiVersions = map[string][]string{
 		//"35.0", // Introduced support
 		"37.0", // Added metadata support
 	},
+	types.OpenApiPathVersion1_0_0 + types.OpenApiEndpointEdgeGateways: {
+		//"35.0", // Introduced support
+		"37.1", // Exposes computed field `UsingIpSpace` in `types.EdgeGatewayUplinks`
+	},
+	types.OpenApiPathVersion1_0_0 + types.OpenApiEndpointIpSpaceUplinksAllocate: {
+		//"37.1", // Introduced support
+		"37.2", // Adds 'value' field
```

Comment:
Did 37.2 introduce a new field that wasn't there before?

Reply:
Yep, this was not available in `37.1` (when it was introduced). This mapping will make sure that `getOpenApiHighestElevatedVersion` picks `37.2` when it is available (being on VCD 10.4.2+)